src/cli/utils/utils.go

```diff
@@ -136,3 +140,21 @@ func splitFoldersIntoContainsAndPrefix(folders []string) ([]string, []string) {
 
 	return containsFolders, prefixFolders
 }
+
+// SendStartCorsoEvent utility sends corso start event at start of each action
+func SendStartCorsoEvent(
+	ctx context.Context,
+	s storage.Storage,
+	tenID string,
+	data map[string]any,
+	repoID string,
+	opts control.Options,
+) {
+	bus, err := events.NewBus(ctx, s, tenID, opts)
+	if err != nil {
+		logger.Ctx(ctx).Debugw("analytics event failure", "err", err)
```

Comment:
Lets do info here, since most users don't run with debug logging.
```suggestion
		logger.Ctx(ctx).Infow("analytics event failure", "err", err)
```

Reply:
The function sending event - https://github.com/alcionai/corso/blob/a5ecc8dd7ea1295a15b9fe15ac4042669f6f6583/src/internal/events/events.go#L150
uses  `Debug` so it would be on similar line if kept debug.

Reply:
Ah: we originally did that because we weren't logging to a file. Now that we're logging to a file, I think it's fine for both to be infos.